server/src/test/java/io/envoyproxy/controlplane/server/EnvoyContainer.java

```diff
@@ -30,6 +30,8 @@
   }
 
   EnvoyContainer(String config, Supplier<Integer> controlPlanePortSupplier, int apiVersion) {
+    // this version is changed automatically by /tools/update-sha.sh:57
```

Comment:
nit: uppercase beginning of each sentence in comment section and end sentence with '.'